modules/oc-by-example-content.adoc

```diff
@@ -0,0 +1,4723 @@
+// NOTE: The contents of this file are auto-generated
+
+[id="openshift-cli_{context}"]
+= OpenShift CLI (oc) commands
+
+
+
+ifdef::openshift-enterprise,openshift-origin[] 
+
+== oc adm build-chain
```

Comment:
Should these be `[discrete]`?

Reply:
No, because we want them to show up in the TOC